source/installation/troubleshooting.markdown

```diff
@@ -0,0 +1,59 @@
+---
+title: "Troubleshooting installation problems"
+description: "Common installation problems and their solutions."
+---
+
+It can happen that you run into trouble while installing and onboarding Home Assistant. This page is here to help you solve the most common problems.
+
+## Can’t access Home Assistant in my browser
+
+### Symptom: “This site can’t be reached”
+
+When trying to access Home Assistant in the browser, the browser shows the message “This site can’t be reached”.
+
+### Description
+
+This means the browser can’t find your Home Assistant installation on the network.
+
+### Resolution
+
+To resolve this issue, try the following steps:
+
+1. Make sure your Home Assistant device is powered up (LEDs are on).
+2. Make sure your Home Assistant installation is connected to the internet:
+   - Make sure the Ethernet cable is plugged-in to both Home Assistant and to your router or switch.
+   - Make sure your network is up and running and your router has internet access.
+     - During installation, Home Assistant needs access to the following URLs:
+       - version.home-assistant.io: for online check and to fetch new version information.
+       - github.com: to update metadata for the add-on store.
+       - ghcr.io: the GitHub container registry to fetch new Home Assistant updates.
+3. Make sure the system on which you opened the browser to access Home Assistant is connected to the same network as Home Assistant.
+   - For example, you might have 2 networks with the same name, but one is on 2.4&nbsp;GHz and the other on 5&nbsp;GHz.
+   - One of your devices might default to another network. Make sure they are in the exact same network.
```

Comment:
Well, typically, if SSID's are separated, they are still connected to the same network. It is rather uncommon that those would be different networks in that sense.

Also not sure what you meant by "One of your devices"....

How about simply checking WiFi, e.g.:
```
   - For example, if the system your Browser runs on is using WiFi, make sure it is using the WiFi which connects to the same network Home Assistant is connected to.
```